lib/modules/accountSwitcher.js

```diff
@@ -336,7 +336,7 @@ async function switchTo(user) {
 		url: '/api/login',
 		data: {
 			user: username,
-			passwd: password,
+			passwd: password ? password : window.prompt(i18n('accountSwitcherPasswordPrompt', username)),
```

Comment:
Let's move this before `otp`, since usually passwords are requested before two-factor, and so logout can happen in the background

Reply:
Sure thing! Is this (6aae1ae) OK?

Reply:
Sure thing! Is this (6aae1ae) OK?